nativeApp/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "twine-native-app",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject",
+    "lint": "eslint .  --ext .ts,.tsx",
+    "test": "jest",
+    "cover": "jest -- --coverage"
+  },
+  "dependencies": {
+    "@types/styled-components": "^4.1.19",
+    "axios": "^0.19.0",
+    "expo": "^34.0.1",
+    "qs": "^6.8.0",
+    "react": "16.8.3",
+    "react-dom": "^16.8.6",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
```

Comment:
er just checking that this is really how we're meant to pull in RN?

Reply:
Yup, this is set by expo when initialised